tests/ArchiveComparatorTests.js

```diff
@@ -1,8 +1,8 @@
-var lib = require("../source/module.js");
+var lib = require("../source/node/index.js");
```

Comment:
Some of the tests also use `var`. its not important but is it easy to change those as well?

Reply:
Maybe these could be part of a later refactor?